build.cake

```diff
@@ -139,6 +139,25 @@ Task("PackageZip")
     Zip(BIN_DIR, File(path), zipFiles);
 });
 
+Task("PackageChocolatey")
+.Does(() =>
+{
+	CreateDirectory("PACKAGE_DIR");
+
+	ChocolateyPack("nunit-project-editor.nuspec", 
+		new ChocolateyPackSettings()
+		{
+			OutputDirectory = PACKAGE_DIR,
+			Files = new []
+			{
+				new ChocolateyNuSpecContent { Source = "LICENSE.txt" },
+				new ChocolateyNuSpecContent { Source = "CHANGES.txt" },
+				new ChocolateyNuSpecContent { Source = BIN_DIR + "nunit-editor.exe", Target = "bin" },
```

Comment:
@CharliePoole since you are embedding the exe in the package, which is fine and encouraged, we would ask that you include a verification.txt. This is used by the moderators and community members to ensure that the contained binary is legitimate. Ideally, this would include the checksums for the files. Here is an example: https://github.com/chocolatey/chocolatey-coreteampackages/blob/b3fa04db54fc2b649959ef94faa1c4e704bd75bb/automatic/qbittorrent/tools/VERIFICATION.txt